clang/test/KeyInstructions/lit.local.cfg

```diff
@@ -0,0 +1,2 @@
+if not config.has_key_instructions:
+    config.unsupported = True
```

Comment:
Is there anywhere deeper in the clang/test directory we can squirrel this -- most of the existing subdirectories are quite substantive pieces of the compiler, which of course debug-info is too, but possibly not key instructions by itself.

(RIP your future tree conflicts).

Reply:
> (RIP your future tree conflicts).

Yeah... 😢 

I went looking for clang/test/DebugInfo, but that did not exist. Our existing debug info IRgen tests are split by language into CodeGen/CodeGenCXX/CodeGenObj, which is not great organization.

I think clang/test/DebugInfo/KeyInstrs would be a good final home for these tests, and organizing the existing tests can be future (never?) work.